valve-monitor/web-app/src/styles/Layout.module.scss

```diff
@@ -5,18 +5,16 @@
   top: 9% !important;
 }
 
-.mainContentWrapper {
+.mainContent {
+  // required to override default Ant Design component styling
+  min-height: calc(100vh - $header-bar-height - $footer-bar-height) !important;
+  width: clamp($page-content-xs, $page-content-lg, calc(100vw - 79px));
+  margin: 0 auto;
   background-color: #fff;
-  padding: 32px;
 
-  .mainContent {
-    // required to override default Ant Design component styling
-    min-height: calc(
-      100vh - $header-bar-height - $footer-bar-height
-    ) !important;
-    width: clamp($page-content-xs, $page-content-lg, calc(100vw - 79px));
-    margin: 0 auto;
-    padding: 20px;
+  :global .ant-row {
+    margin-left: 0 !important;
+    margin-right: 0 !important;
```

Comment:
Feel free to ignore this, but it does feel weird to globally kill all left/right margins for all rows globally. This makes more sense to do in the Home module. Although again, very very trivial.

Reply:
I did that to eliminate a weird horizontal scroll bar that showed up in the mobile view injected by `.ant-row`